app/controllers/application_controller.rb

```diff
@@ -7,4 +7,14 @@ class ApplicationController < ActionController::Base
 
   responders :flash
   respond_to :html
+
+  expose :serialized_current_user_data, -> { serialized_current_user_data }
+
+  private
+
+  def serialized_current_user_data
+    return nil unless user_signed_in?
+
+    UserSerializer.new(current_user).to_json
+  end
```

Comment:
```suggestion
  def serialized_current_user_data
    UserSerializer.new(current_user).to_json if user_signed_in?
  end
```

Reply:
I think "Explicit is better than implicit." I prefer to explicitly return values to easy see where and what I''ll get)